tests/src/test/java/alluxio/client/fuse/AbstractFuseIntegrationTest.java

```diff
@@ -106,6 +106,7 @@ public static void beforeClass() {
 
   @Before
   public void before() throws Exception {
+    Configuration.set(PropertyKey.USER_METADATA_CACHE_MAX_SIZE, 0);
```

Comment:
One of the case is: PropertyKey.USER_METADATA_CACHE_MAX_SIZE = 0;
What about the other case where PropertyKey.USER_METADATA_CACHE_MAX_SIZE is some other value, e.g. 1000.

Reply:
Our current Fuse tests focus on the case without any cache.
We do have metadata cache test tests the functionality of metadata  cache, similarly, we have data cache tests covers the functionality of client side data cache.

Good call that in the future can include more fuse tests covering the test cases with data/metadata cache.

If the metadata cache max size is bigger than zero, then metadata cache is enabled.

Reply:
OK